### ISSUE TITLE
Arc Control not responding to specific combination of hotkeys

### ISSUE THREAD

**Issue by derlade** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#235]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Intel Arc Control

### Processor / Processor Number [Required]

Intel Core i3-12100

### Graphic Card [Required]

Intel Arc A770 LE

### GPU Driver Version [Required]

- [X] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034
- [ ] 31.0.101.4032
- [ ] 31.0.101.3975
- [ ] 31.0.101.3959
- [ ] 31.0.101.3802
- [ ] 31.0.101.3793
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]

- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [X] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/10355765/igcit_ssu.txt)

### Description and steps to reproduce [Required]

There is a weird hotkey inconsistency with the overlay(s) in Arc Control. 

- When pressing Alt+I or Alt+O FULLY (so full release off all key presses, then Alt, and then I or O while still holding Alt), the respective overlays open normally (hotkey combinations are the default ones). 
- When pressing AND holding Alt and pressing I or O multiple times, while still holding Alt from the first key press, the overlays also open normally. 
- BUT: When opening the main overlay by pressing Alt+I, releasing the key combination, and THEN pressing and holding Alt while pressing I AND THEN either I or O (while still holding Alt), the overlays won't open until Alt is released.
- This doesn't happen when repeating holding Alt from the telemetry overlay. 

It seems like holding Alt after closing the overlay bugs out the continuous key press in some fashion. I have uploaded the ssu from my previous post; if a more updated one is needed, I will provide it ASAP.

**Comment by IntelSupport-Arun** [comment]
Hi @derlade ,
I'm a bit confused after reading the steps you mentioned in the issue description.
Could you please provide me with a step-by-step issue reproduction process that I can follow and determine if I can reproduce the issue?

**Comment by derlade** [comment]
Hi @IntelSupport-Arun,

1. Open the Arc Control overlay by pressing Alt+I.
2. Release the key combination.
3. Hold Alt.
4. Press I, still keep holding Alt.
5. While still holding Alt, try pressing I or O. Neither the Arc Control overlay nor the telemetry overlay will open, until you release Alt from Step 3 and reissue the full key combination.

Hope this helps.

**Comment by IntelSupport-Arun** [comment]
Hi @derlade ,
Will the Arc Control overlay work if you release the ALT after step 4 and try again the key combination ALT+I/O?

**Comment by derlade** [comment]
Hi @IntelSupport-Arun,

yes, it will. It is only after this specific arrangement of key combinations that the overlays won't work; any other combo works.

**Comment by IntelSupport-Arun** [comment]
Hi @derlade , 
I am able to reproduce the same observation on our side as well. Based on the observation I have escalated the case to next level for further verification and debug. Will update you once I hear from them. 
**Issue Verified and confirmed on the below config:**
Platform checked : Intel Core i7 12700K/ AMD Ryzen 7 5800X 
Discrete Graphics Card: Arc A770(16Gb)
Latest Graphics driver: 31.0.101.4091 & 31.0.101.4123
CI Master Driver Version: gfx-driver-ci-master-13231
Intel Arc Control Panel Version: 1.64.4583.3

**Comment by EstebanIntel** [comment]
Hi @derlade, I followed up with our support team on this issue and they mentioned that the app was developed expecting full key presses and releases. So, this been a very edge case, it will be tagged for future work, as the limited development resources are been assigned to issues with higher priority.

We appreciate you testing our app and reporting this issues. But for the time been, we cannot give a target date to when we might work on this specific case.
